t/io/utf8.t

```diff
@@ -83,21 +83,21 @@ close(F);
   }
 
     { # Check byte length of $b
-	use bytes; my $y = length($b);
-	cmp_ok( $y, '==', 1 );
+    use bytes; my $y = length($b);
```

Comment:
something changed the indentation?

Reply:
I'm not sure what happened here.  There are places where it makes sense to standardize on spaces (usually 4) rather than tabs for a particular block.

Reply:
not a big deal